napi-inl.h

```diff
@@ -1267,7 +1257,7 @@ inline Reference<T>::operator napi_ref() const {
 
 template <typename T>
 inline bool Reference<T>::operator ==(const Reference<T> &other) const {
-  HandleScope scope = HandleScope(Env());
+  HandleScope scope = Handlescope(_env);
```

Comment:
Huh, good catch. That shouldn't compile... but that's a funny thing about inline functions, they don't get compiled if nothing uses them.